### ISSUE TITLE
12/6 logistics artifact; Reconnaissance Interdiction did not port the enemy force to the Objective Status screen on scenario resolution

### ISSUE THREAD

**Issue by OrbMonky** [MegaMek/mekhq#5342]
### Prerequisites and Pre-Issue Checklist

- [X] I'm reporting the issue to the correct repository:
- [MegaMek](https://github.com/MegaMek/megamek/issues)
- [MegaMekLab](https://github.com/MegaMek/megameklab/issues)
- [MekHQ](https://github.com/MegaMek/mekhq/issues)

- [X] I've tested the issue against at least the latest MILESTONE version
- [X] I've asked on the MegaMek Discord about the error
- [X] I've reviewed the BattleTech rules and MekHQ documentation, and I've confirmed that something isn't working as intended.
- [X] I've searched the Github tracker and haven't found the issue listed

### Severity *

Low (Minor/Nuisance): Minor glitches or cosmetic issues that don’t affect gameplay and occur rarely.

### Brief Description *

![image](https://github.com/user-attachments/assets/5502e754-ad08-4f12-acfd-19b8c87c1f04)
[logs.zip](https://github.com/user-attachments/files/18115934/logs.zip)
logs should include mm/mhq logs, campaign, customs, save before and after in campaign, and autosaves of mm game

No enemy forces listed for completion, though the objective was available and so a mostly natural and correct conclusion could be reached as long as you knew the results.

### Operating System *

Windows 10

### Java Version *

17

### MekHQ Suite Version *

None

### Custom MekHQ Version

12/6 logistics artifact

### Final Checklist

- [X] I've checked to make sure that this issue has not already been filed
- [X] I'm reporting only one issue in this ticket for clarity and focus

**Comment by IllianiCBT** [comment]
Nothing in the log, and I've been unable to replicate. What appears to have happened is that, for whatever reason, the objectives failed to parse the OpFor. Reading them as being, essentially, empty. I'm unable to see anything in the associated xml out of order, so I think this just might be a transient error that we'll need to keep an eye on.

**Comment by IllianiCBT** [comment]
Out of curiosity, did you regenerate or edit bot forces at all?

**Comment by OrbMonky** [comment]
No, I have not generated bot forces on this campaign, so it should all be stock 'what mekhq wanted' bot forces/etc. This *IS* however a hand crafted contract done through Add Mission, which may be the source of some of this funkiness. Prior generated contracts seemed to work mostly fine.

**Comment by IllianiCBT** [comment]
Keep an eye on this, and if you notice it happening again try to remember every step you took to reach that point. Leave nothing out.

**Comment by IllianiBird** [comment]
This shouldn't be an issue any longer. Closing.
